README.md

```diff
@@ -3,13 +3,13 @@
 Docker files for [Deno](https://github.com/denoland/deno) published on
 Dockerhub:
 
-- Alpine Linux: [denoland/deno:alpine](https://hub.docker.com/r/denoland/deno)
-- Centos: [denoland/deno:centos](https://hub.docker.com/r/denoland/deno)
-- Debian: [denoland/deno:debian](https://hub.docker.com/r/denoland/deno)
-  (default)
-- Distroless: [denoland/deno:distroless](https://hub.docker.com/r/denoland/deno)
-- Ubuntu: [denoland/deno:ubuntu](https://hub.docker.com/r/denoland/deno)
-- Only the binary: [denoland/deno:bin](https://hub.docker.com/r/denoland/deno)
+-   Alpine Linux: [denoland/deno:alpine](https://hub.docker.com/r/denoland/deno)
+-   Centos: [denoland/deno:centos](https://hub.docker.com/r/denoland/deno)
+-   Debian: [denoland/deno:debian](https://hub.docker.com/r/denoland/deno)
+    (default)
+-   Distroless: [denoland/deno:distroless](https://hub.docker.com/r/denoland/deno)
+-   Ubuntu: [denoland/deno:ubuntu](https://hub.docker.com/r/denoland/deno)
+-   Only the binary: [denoland/deno:bin](https://hub.docker.com/r/denoland/deno)
```

Comment:
Prettier again? I wonder why it does this.

Reply:
Every time 😭